### ISSUE TITLE
Sending a packet to the Multicast Address with multiple interfaces causes Segmentation fault

### ISSUE THREAD

**Issue by benpicco** [RIOT-OS/RIOT#11980]
#### Description
On RIOT, if there is more than once network interface, `ping6 ff02::1` will cause a segmentation fault.

#### Steps to reproduce the issue
In `examples/gnrc_networking`:

```
$ ../../dist/tools/tapsetup/tapsetup
$ sed -i '1i#define NETDEV_TAP_MAX 2' ../../cpu/native/include/netdev_tap_params.h
$ make GNRC_NETIF_NUMOF=2
$ bin/native/gnrc_networking.elf tap0 tap1
> ping6 ff02::1

Program received signal SIGSEGV, Segmentation fault.
0x5656dc02 in _fill_ipv6_hdr (netif=0x5659dcd8 <_netifs+312>, ipv6=0x5659de68 <_pktbuf+40>)
    at /home/benpicco/dev/RIOT/sys/net/gnrc/network_layer/ipv6/gnrc_ipv6.c:308
308	    hdr->len = byteorder_htons(gnrc_pkt_len(ipv6->next));
(gdb) p hdr
$1 = (ipv6_hdr_t *) 0x40
```

#### Expected results
The packet should be send over both interfaces.
Also, `ipv6->data` should point to a valid address.

#### Actual results
RIOT crashes. This happens both on native and on actual hardware.

#### Versions
RIOT master (ca2b00875a9e91458ba361e293cc4e97d281ecf8)

**Comment by miri64** [comment]
I was also able to reproduce this with a `netdev_tap` interface and a `socket_zep` interface.

**Comment by miri64** [comment]
This slipped somehow under my radar and I only noticed it by looking at the issue list for the current release milestone. This is an easy fix, I will pull request ASAP.

**Comment by miri64** [comment]
Fixed in #12512.

**Comment by miri64** [comment]
Still not fixed for the release.

**Comment by miri64** [comment]
Fixed by #12521.
